docs/get-started/lotus/installation.md

```diff
@@ -311,19 +335,29 @@ We recommend that macOS users use [Homebrew](https://brew.sh) to install each of
 
    This is due to a Lotus bug that prevents Lotus from running on a processor without `adx` instruction support, and should be fixed soon.
 
-1. Build Lotus:
+5. Build and install Lotus
```

Comment:
Regarding list number, using `1.` for each item is a way to write ordered lists in Markdown. This supports easier re-numbering in the future, if necessary. Let's preserve that since we use it across the repo.

More info: https://www.markdownguide.org/basic-syntax/#ordered-lists

Reply:
Ok, I will update.